legacy/src/api/assets/tokens.md

```diff
@@ -68,7 +66,7 @@ Token value may be set in multiple currencies and is the same for all tokens in
 
 {% h4 Error Responses %}
 
-| Status |              Code               |                             Description                              |
-| :----- | :------------------------------ | :------------------------------------------------------------------- |
-| 403    | TOKEN_ALREADY_CREATED           | Token with supplied parameters already exists.                       |
-| 403    | ACCOUNT_LIVENESS_MISMATCH       | The account is test and the supplied liveness is main or vice versa. |
+| Status |                 Code                 |                               Description                                |
+| :----- | :----------------------------------- | :----------------------------------------------------------------------- |
+| 403    | TOKEN_ALREADY_CREATED                | Token with supplied parameters already exists.                           |
+| 403    | COLLECTION_ACCOUNT_LIVENESS_MISMATCH | The account is test and the collection's liveness is main or vice versa. |
```

Comment:
Could just use LIVENESS_MISMATCH? Follows the pattern used for Merchants and Batches

Reply:
Yep, can do nice one!